backend/src/nodes/nodes/pytorch/convert_to_onnx.py

```diff
@@ -64,4 +85,6 @@ def run(self, model: PyTorchModel) -> OnnxModel:
             f.seek(0)
             onnx_model_bytes = f.read()
 
-        return OnnxModel(onnx_model_bytes)
+        fp_mode = "fp16" if fp16 else "fp32"
```

Comment:
Why does `fp16` and not `should_use_fp16` determine `fp_mode`? Right now, the model could be fp32 with `fp_mode == "fp16"`.

Reply:
I don't think it could because the assert statement up top would throw an error, but you are correct it should be `should_use_fp16` anyway